tests/config.json

```diff
@@ -22,6 +22,9 @@
         "drag": "http://localhost:3456/drag.html",
         "queryString": "http://localhost:3456/querystring.html",
         "waitAndClick": "http://localhost:3456/wait_and_click.html",
+        "waitAndType": "http://localhost:3456/wait_and_type.html",
```

Comment:
As you said before. It would be nice to merge all these html files to reduce the number of testing files if you agree (Not a requirement either)